libraries/AP_LandingGear/AP_LandingGear.cpp

```diff
@@ -82,6 +91,12 @@ AP_LandingGear *AP_LandingGear::_singleton;
 /// initialise state of landing gear
 void AP_LandingGear::init()
 {
+    if (!_enable.configured() && (SRV_Channels::function_assigned(SRV_Channel::k_landing_gear_control) || 
+            (_pin_deployed > 0) || (_pin_weight_on_wheels > 0))) {
+        // if not configured set enable param if output servo or sense pins are defined
+        _enable.set_and_save(1);
```

Comment:
I see, this is a bit like a parameter conversion in that you want to get that parameter set for existing users.

Reply:
Since there is a init function already it seemed a shame to go and do dedicated param conversion in both plane and copter. Your right in that this may mean that if the user loads params very soon after boot the first time the param may not have been set yet. But that will only happen once and a param refresh and/or a reboot will fix.